### PR TITLE
typo fix in staging/.../apiserver/*

### PR DESCRIPTION
**Pull request by CKchen0726**
What type of PR is this?
/kind cleanup

What this PR does / why we need it:
typo fix in staging/.../apiserver/*

Which issue(s) this PR fixes:

Special notes for your reviewer:

Does this PR introduce a user-facing change?
```release-note
NONE
```

Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:
```release-note
NONE
```

### CODE REVIEW

**Comment by kwiesmueller** [comment]
/sig auth
/assign @apelisse 
could you give the remaining approvals?

File: staging/src/k8s.io/apiserver/pkg/authorization/union/union.go [hunk 1]
```diff
@@ -44,18 +44,18 @@ func New(authorizationHandlers ...authorizer.Authorizer) authorizer.Authorizer {
 // Authorizes against a chain of authorizer.Authorizer objects and returns nil if successful and returns error if unsuccessful
 func (authzHandler unionAuthzHandler) Authorize(ctx context.Context, a authorizer.Attributes) (authorizer.Decision, string, error) {
 	var (
-		errlist    []error
-		reasonlist []string
+		errList    []error
+		reasonList []string
```
Comment (liggitt) [inline staging/src/k8s.io/apiserver/pkg/authorization/union/union.go:48]: Thanks for the PR, but renaming local private variables like this is change without much benefit. I'd suggest reverting this file and limiting this PR to the typo fix.

**Comment by dims** [comment]
@CKchen0726 please see feedback from reviewer above

**Pull request closed without merging**